feat : add test cases with using jest framework

pooja-ces opened the pull request.

codecrafter-a committed:
feat : add test cases with using jest framework

vaishali3012 commented on src/providers/staticData.test.ts:
```diff
@@ -0,0 +1,118 @@
+        });
+    });
+
+    test('getExternalService should return the expected external services data', async () => {
```
can you try using it

vaishali3012 commented on src/providers/staticData.test.ts:
```diff
@@ -0,0 +1,118 @@
+describe('staticDataProvider', () => {
+    test('getServices should return the expected services data', async () => {
+        const services = await staticDataProvider.getServices();
+        expect(services).toBeDefined();
```
separate assert act and arrange

pooja-ces committed:
fix :-
=> I have test cases test using it
=> I have fix separate assert act and arrange
